src/odemis/gui/cont/features.py

```diff
@@ -189,9 +191,14 @@ def _on_ctrl_feature_z_change(self):
         Get the current feature Z ctrl value to set feature Z position
         :return: (tuple of 3 floats) the full position including the Z ctrl value
         """
+        # HACK: sometimes the event is first received by this handler and later
+        # by the UnitFloatCtrl. So the value is not yet computed => Force it, just in case.
+        self._panel.ctrl_feature_z.on_text_enter(None)
+        zpos = self._panel.ctrl_feature_z.GetValue()
```

Comment:
I assume you checked this but is it correct that the call to this variable/attribute changed from Value --> GetValue()?

Reply:
Yes, that's on purpose. Because on this control .Value != .GetValue().